### DISCUSSION TITLE
Webview2 not installed on 1/2 of our Lab systems that are managed via SCCM

### DISCUSSION THREAD

**Discussion by scarneol** [MicrosoftEdge/WebView2Feedback#3887] [Q&A]
I am trying to figure out how Webview2 gets installed in the first place, how it us maintained and why I only see it installed on 675 or our 1600+ lab systems all running the same software and all managed exactly the same way with Configuration Manager.

**Comment by champnic** [comment]
Hey @scarneol - The WebView2 Runtime is typically deployed by apps during install. It will be included inbox with Windows 11, but on Windows 10 and lower will need to be deployed. You can find more info on this here: https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution

On machines in the wild if WebView2 Runtime is already on the machine that means that a different app has already installed it. For example, new versions of Office are deploying WebView2 Runtime. It's unclear to me why your lab systems are exhibiting differences, except that maybe different apps have been run at different times? Or some apps have chose to update and run the install while others haven't?
